assets/js/components/shapeshift/shift-confirm.component.js

```diff
@@ -26,7 +27,7 @@ function ShiftConfirmController (AngularHelper, $scope, Exchange, Wallet, Ethere
   $scope.to = $scope.quote.toCurrency === 'btc' ? Wallet.getDefaultAccount() : Ethereum.defaultAccount;
   $scope.fromCurrency = $scope.quote.fromCurrency === 'btc' ? $scope.bitcoin : $scope.ether;
   $scope.total = parseFloat($scope.quote.depositAmount) + parseFloat($filter('convert')(this.fee, $scope.fromCurrency, false));
-  $scope.onExpiration = () => $scope.lock();
+  $scope.onExpiration = () => { this.onExpire(); };
```

Comment:
This is fine, but you can save code by changing on-expire and onExpire to be onExpiration(), since you're already calling that in confirm.pug and you don't need to call anything else.